app/src/main/java/com/sofascore/scoreandroidacademy/ui/adapter/EventDetailsAdapter.kt

```diff
@@ -0,0 +1,228 @@
+
+            when (incident.type) {
+                "goal" -> {
+                    when (incident.goalType) {
+                        "regular" -> logoType.setImageResource(R.drawable.ic_football_regular_goal)
+                        "penalty" -> logoType.setImageResource(R.drawable.ic_football_penalty_score)
+                        "penaltymissed" -> logoType.setImageResource(R.drawable.ic_football_penalty_missed)
+                        "owngoal" -> logoType.setImageResource(R.drawable.ic_football_autogoal)
+
+                        "onepoint" -> logoType.setImageResource(R.drawable.ic_basketball_1)
+                        "twopoint" -> logoType.setImageResource(R.drawable.ic_basketball_2)
+                        "threepoint" -> logoType.setImageResource(R.drawable.ic_basketball_3)
+
+                        "touchdown" -> logoType.setImageResource(R.drawable.ic_amf_touchdown)
+                        "extrapoint" -> logoType.setImageResource(R.drawable.ic_amf_extra_point)
+                        "fieldgoal" -> logoType.setImageResource(R.drawable.ic_amf_field_goal)
+                        "safety" -> logoType.setImageResource(R.drawable.ic_amf_rogue)
+
+                    }
+                }
+                "card" -> {
+                    when (incident.color) {
+                        "yellow" -> logoType.setImageResource(R.drawable.ic_football_card_yellow)
+                        "red" -> logoType.setImageResource(R.drawable.ic_football_card_red)
+                        "yellowred" -> {
+                            val yellowCardDrawable = ContextCompat.getDrawable(
+                                context,
+                                R.drawable.ic_football_card_yellow
+                            )
+                            val redCardDrawable = ContextCompat.getDrawable(
+                                context,
+                                R.drawable.ic_football_card_red
+                            )
+
+                            val layers = arrayOf(yellowCardDrawable, redCardDrawable)
+                            val layerDrawable = LayerDrawable(layers)
+
+                            layerDrawable.setLayerInset(1, 12, 8, 0, 0)
+
+                            logoType.setImageDrawable(layerDrawable)
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RecyclerView.ViewHolder {
+        return when(viewType){
+            ViewType.LayoutOne -> EventStatusViewHolder(LayoutEventDetailsMatchStatusBinding.inflate(LayoutInflater.from(parent.context), parent, false))
+            ViewType.LayoutTwo -> EventIncidentViewHolder(LayoutFootballHomeIncidentsBinding.inflate(LayoutInflater.from(parent.context), parent, false))
+            ViewType.LayoutThree -> EventIncidentAwayViewHolder(LayoutFootballAwayIncidentsBinding.inflate(LayoutInflater.from(parent.context), parent, false))
```

Comment:
binding mozes bindati i na drugi layout ako ima iste id-jeve viewova
tako da mislim da bi mogao pisati ovako nesto i oba viewholdera predati npr. `LayoutFootballHomeIncidentsBinding` samo je jednom to home layout a drugom away layout:
```suggestion
             ViewType.LayoutTwo -> {
                val binding = LayoutFootballHomeIncidentsBinding.inflate(LayoutInflater.from(parent.context), parent, false)
                EventIncidentViewHolder(binding)
            }
            ViewType.LayoutThree -> {
                val layout = LayoutFootballAwayIncidentsBinding.inflate(LayoutInflater.from(parent.context), parent, false).root
                val binding = LayoutFootballHomeIncidentsBinding.bind(layout)
                EventIncidentAwayViewHolder(binding)
            }
```

Reply:
also trebas koristiti neka smislenija imena, da se odmah zna o cem je rijec, a ne LayoutOne, LayoutTwo etc.